src/desktop/models/article.coffee

```diff
@@ -247,7 +247,7 @@ module.exports = class Article extends Backbone.Model
 
   # article metadata tag for parse.ly
   toJSONLD: ->
-    tags = @get('tags')
+    tags = @get('tags') || []
```

Comment:
This was breaking some of the JSONLD tests because they couldn't handle an undefined array -- was resulting in errors in promises that caused tests to silently break.